.github/workflows/build-test.yml

```diff
@@ -143,7 +142,7 @@ jobs:
   coverage-and-docs:
     # prevent stuck jobs consuming runners for 6 hours
     timeout-minutes: 60
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     needs: pre-flight-check
     steps:
       - uses: actions/checkout@v3
```

Comment:
Nothing for this PR but at some time we should cache the coverage result and move the upload to a separate job. Yesterday I had to rerun this job 3 times until the upload worked.